src/current/_includes/cockroachcloud/first-org-user-roles.md

```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-The user who creates a new organization is assigned a combination of Org Administrator, Billing Coordinator, and Cluster Admin at the organization scope. Any of these roles may subsequently be removed, although another user must have the Org Administrator role and the Cluster Admin role at the organization scope, before either of those can be removed. This is to ensure that at least one user has each of these roles.
+The user who creates a new organization is assigned the Org Administrator, Billing Coordinator, Folder Administrator, and Cluster Admin roles at the organization scope. Any of these roles may subsequently be removed by a user with the Org Administrator role and the Cluster Admin role at the organization scope. This is to ensure that at least one user has each of these roles.
```

Comment:
since we are mentioning all these roles suggest adding a link to some docs for what these roles are / where I can read more about them